### PR TITLE
Add funny funnies.py

### PR DESCRIPTION
**Pull request by Jocelyn-JE**
haha funi 9/11

### CODE REVIEW

**Comment by Jocelyn-JE** [comment]
plz merge pull request

File: cogs/funnies.py [hunk 1]
```diff
@@ -53,6 +53,9 @@ async def on_message(self, message: discord.Message):
 
         if message.content == "dc":
             await message.channel.send("https://autocompressor.net/av1?s=wfinqwL9")
-        
+
+        if self.bot.mogi["status"] and "9/11" in message.content.lower():
```
Comment (probablyjassin) [inline cogs/funnies.py]: please also add a check to make sure the amount of players in the mogi is actually 11

Comment (Kevnkkm) [inline cogs/funnies.py]:
``` suggestion
        if self.bot.mogi["status"] and len(self.bot.mogi["players"]) == 9 and "9/11" in message.content.lower():
```
What jassin said

**Review by Kevnkkm — CHANGES_REQUESTED** [review]
please fix

**Review by Kevnkkm — APPROVED** [review]
good guy

**Pull request merged**